### PR TITLE
[CloudWatchLogs] Add CreateLogStream

### PR DESCRIPTION
**Pull request by Amunak**
This is similar to #957, adding CreateLogStream as it's kinda missing.

I have no idea if I've written the tests (or anything else) correct, but they should be right, I think? Feel free to edit anything.

Hope to get this feature soon;
Thanks.

### CODE REVIEW

**Comment by Amunak** [comment]
I have no idea what exactly is failing there (or why) :/

File: src/Service/CloudWatchLogs/tests/Unit/Input/CreateLogStreamRequestTest.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+            'logGroupName' => 'my-log-group',
+            'logStreamName' => 'my-log-stream',
+        ]);
+
+        // see https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_CreateLogStream.html
+        $expected = '
+POST / HTTP/1.0
+Content-Type: application/x-amz-json-1.1
+X-Amz-Target: Logs_20140328.CreateLogStream
+{
```
Comment (jderusse) [inline src/Service/CloudWatchLogs/tests/Unit/Input/CreateLogStreamRequestTest.php:22]: This should be a valid HTTP response: headers and body are separated by a blank line

```suggestion
X-Amz-Target: Logs_20140328.CreateLogStream

{
```

Comment (Amunak) [reply src/Service/CloudWatchLogs/tests/Unit/Input/CreateLogStreamRequestTest.php]: Oh I'm dumb, thanks.

**Comment by jderusse** [comment]
thank you @Amunak

**Comment by Amunak** [comment]
Thank you for the fast merge!

How do the releases for the sub-packages work, do they happen automagically sometime soon after the merge or do you have to trigger them manually?

**Pull request merged**